encrypted-media-respec.html

```diff
@@ -2401,7 +2404,7 @@
                 </li>
                 <li>
                   <p>
-                    Run the <a def-id="update-key-statuses-algorithm"></a> algorithm, passing <var>session</var> and <var>statuses</var>.
+                    <a def-id="Queue-a-task"></a> to run the <a def-id="update-key-statuses-algorithm"></a> algorithm, passing <var>session</var> and <var>statuses</var>.
```

Comment:
Already in a task (line 2395), so this is unnecessary, though there are no negative side-effects of doing so.